paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/sink/cdc/RichCdcMultiplexRecord.java

```diff
@@ -49,6 +49,10 @@ public RichCdcMultiplexRecord(
         this.cdcRecord = cdcRecord;
     }
 
+    public String databaseName() {
+        return databaseName;
+    }
+
```

Comment:
Seems no class call this?

Reply:
call here:

https://github.com/apache/incubator-paimon/blob/e816c51eea232d163c4985b1a0dd94a7011164c9/paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/sink/cdc/RichCdcMultiplexRecordEventParser.java#L87-L89